src/fprime/cookiecutter_templates/cookiecutter-fprime-deployment/{{cookiecutter.deployment_name}}/Top/topology.fpp

```diff
@@ -101,6 +101,7 @@ module {{cookiecutter.deployment_name}} {
       rateGroup1.RateGroupMemberOut[0] -> tlmSend.Run
       rateGroup1.RateGroupMemberOut[1] -> fileDownlink.Run
       rateGroup1.RateGroupMemberOut[2] -> systemResources.run
+      rateGroup1.RateGroupMemberOut[3] -> comQueue.run
```

Comment:
This line is the only actual change in this PR; the other changes were merged aas of https://github.com/nasa/fprime-tools/pull/250/files

I can never tell why GitHub doesn't pick those things up sometimes